### ISSUE TITLE
Crash when going on Audio/Video tab

### ISSUE THREAD

**Issue by gustavklopp** [qTox/qTox#2901]
Lubuntu 15.10
"You are using qTox version v1.2-334-g7a846b8."

When trying to go on the tab Audio/Video: crash of qTOX

The console shows (I've launched qTox from the console):

```
[12:07:00.394] src/audio/audio.cpp:267 : Debug: Opening audio output "Audio interne Stéréo analogique"
[12:07:00.398] src/audio/audio.cpp:292 : Debug: Opened audio output "Audio interne Stéréo analogique"
[12:07:00.430] src/widget/form/settings/avform.cpp:143 : Warning: Invalid mode index
[12:07:00.430] src/video/camerasource.cpp:282 : Debug: Closing device  "none"
[12:07:00.430] src/video/camerasource.cpp:209 : Debug: Opening device  "/dev/video0"
[video4linux2,v4l2 @ 0xce58ea0] Cannot find a proper format for codec 'mjpeg' (id 8), pixel format 'none' (id -1)
Assertion *codec_id != AV_CODEC_ID_NONE failed at libavdevice/v4l2.c:821
Abandon (core dumped)
```

It's a USB webcam Quickcam Logitech Pro 4000 (Working fine with Cheese)

**Comment by gustavklopp** [comment]
The issue seems related to [#issue2852](https://github.com/tux3/qTox/issues/2852) with the same exact error message (except it's not an error of video upside down but a crash at opening the tab audio/video)

**Comment by tux3** [comment]
This issue should be fixed by #2867, which isn't merged yet

**Comment by gustavklopp** [comment]
I think finally it's a problem Ubuntu (at least 15.10) related. I've run it on Arch Linux and it's working without this crash. 
My bet is that it's due to ffmpeg which doesn't seem always well integrated in Ubuntu... I may be wrong

EDIT: I've found a strange work around for this: I delete the entire ~/.config/qtox (or keep only username.tox and paste it again). And I'm able now to use this USB webcam.

**Comment by paly2** [comment]
I have the same problem...

@gustavklopp :

> EDIT: I've found a strange work around for this: I delete the entire ~/.config/qtox (or keep only username.tox and paste it again). And I'm able now to use this USB webcam.

I tested this, and this works! But I have to delete the files before each starting of qTox and configure the webcam again :/

**Comment by zetok** [comment]
@paly2 could you see whether deleting only `qtox.log` helps?

**Comment by paly2** [comment]
Deleting only `qtox.log` doesn't work, but deleting only `qtox.ini` works, and more precisely the last line of `qtox.ini` (`videoDev`).

**Comment by sudden6** [comment]
This should be fixed with 3f89cd95d5399814461f5dae8d7e2a524637988e
